### ISSUE TITLE
[1.15.2] Alpha 0.13.548 - Placing Supply Ship Creates Dead Area

### ISSUE THREAD

**Issue by Slashaar** [ldtteam/minecolonies#6336]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.15.2
- MineColonies Version: Alpha 0.13.548
- Structurize Version: Alpha 0.13.105
- Related Mods and their Versions: N/A

## Expected behavior
Place Ship and break ship and pick up resources.

## Actual behavior
Entire chunk where 'invisibile anchor block' (for lack of a better description -red wireframe box-) is located creates a dead zone where players are unable to pick up items. Tested only with Acacia Supply Ship.

## Steps to reproduce the issue
1. PReview Acacia Supply ship via free supply ship item.
2. Locate Red Wire frame box and which chunk it is in.
3. Finalize Placement.
4. Break any block or drop any item and you will not be able to pick it up, this includes EXP. 
         -
[2020-04-14-2.log.gz](https://github.com/ldtteam/minecolonies/files/5760100/2020-04-14-2.log.gz)
If you are able to remove the items from the chunk, they will become able to be picked up again.

### Logs

- latest.log: https://gist.github.com/Slashaar/6816a199f4a59dddc84aa60adc4c7c27
- crashlog: n/a

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Slashaar** [comment]
UPDATE: I recreated with a Dark Oak Supply ship. It's anything that falls at or below water level that becomes unable to be picked up.

**Comment by Slashaar** [comment]
![image](https://user-images.githubusercontent.com/66047321/103456079-7a087480-4cb8-11eb-88a2-0cf2ace5f178.png)
Appears to be dead right on top of water too. However one block above water I am still able to pickup.

**Comment by Slashaar** [comment]
![image](https://user-images.githubusercontent.com/66047321/103456121-de2b3880-4cb8-11eb-91a2-e6751359db37.png)
Seems not to correspond with the red wireframe after all. Seems like it's putting a protection on the area starting at one block above waterline and creating a no-pickup dead zone.

**Comment by Raycoms** [comment]
Can you reproduce this only with Minecolonies and no other mods or similar?

**Comment by Slashaar** [comment]
I will test, but I also found the issue in a chunk that has not been built in with MC. It is inside the Town border though.
Again though it starts at Y 63 where it no longer is able to pick up items.

**Comment by Slashaar** [comment]
Tested with just Structurize and Mine Colonies. This does not appear to be specific to Mine Colonies. My apologies. I'm not sure if its an issue with it itself or in combination with another mod. As currently no other mod that would effect chunks is in place in the chunks we've observed the issue.

FWIW:
Server Info where problem occurs.
Minecraft v1.15.2
Forge v31.2.47
Mods installed:
-Security Craft v1.8.20.2
-Angel Ring v1.2.0
-AutoRegLib  v1.5-40
-Biomes O' Plenty v10.0.0.366
-Bonsai Trees v2.1.2.6
-Bookshelf v5.6.40
-C.O.M.B.A.T. v4.9.4
-Chicken Chunks v2.5.1.79
-Clumps v5.0.2.6
-Code Chicken Library v3.3.4.402
-Compact v3.0
-Cucumber v3.0.9
-Curios-Forge v2.0.2.6
-Embellishcraft v2.1.1
-Environmental Creepers v1.6.1
-Extra Disks v1.2.2
-Fast Leaf Decay v22.1
-Flux Networks v5.0.3-4
-Gecko Library v3.0.0
-Voxel Map v1.9.28
-Forgiving Void v4.0.0
-MrCrayFish's Furniture Mod v7.0.0-pre16
-HWYLA v1.10.8-B72
-Industrial Foregoing v2.3.3-e356e61
-Ironchest v10.0.3
-JEI v6.0.3.15
-KleeSlabs v8.0.0
-Kottle v1.5.2
-Libnonymous v1.1.1.5
-MA Chat v3.1.0
-MA Essentials v3.1.1
-Mapper Base v1.1.0.0
-MC Doom v3.6.2
-MCJTY Library v4.1.13
-Macaw Doors v1.0.1fix
-Mekanism v9.10.9.422
-Mekanism Additions v9.10.9.422
-Mekanism Generators v9.10.9.422
-Mekanism Tools v9.10.9.422
-Mine Colonies v0.13.548-Alpha
-Mixin Bootstrap v1.0.5
-MobGrinder v1.4
-Morpheus v4.2.50
-Mystical Aggriculture v3.0.10
-Mystical Aggraditions v3.0.1
-Nature's Compadd v1.8.5
-No Increasing Repair Cost v1.0.0
-Obfuscate v0.5.0
-Performant v3.14m
-Polymorph v0.9
-Project Essentials Core v2.1.0
-Project Essentials Backup v2.1.0
-PTC v31.1.43
-Quark vr2.1-245
-Refined Storage v1.8.8
-RF Tools Base v1.1.10
-RF Tools Builder v2.1.16
-RF Tools Control v3.0.9
-RF Tools Power v2.2.6
-RF Tools Storage v1.1.13
-RF Tools Utility v 2.1.18
-Ring of Repair v1.1.33
-Roadstuff v3.2.0
-Simple Farming v1.3.4
-Solar Flux Reborn v15.2.3
-Storage Drawers v7.0.3
-Storage Tech vR7.2
-Structurize v0.13.105 Alpha
-Titanium v2.4.2
-Tombstone v4.6.0
-Torchmaster v2.2.2
-Trash Slot v11.0.0
-U Team Core v3.0.2.169
-Union Library v1.2.3
-Useful Backpacks v1.10.3.77
-WAWLA v3.0.4
-World Edit v7.1.0
